pkg/validations/validations.go

```diff
@@ -58,7 +58,10 @@ func ValidateHostname(name string) error {
 		return errors.Wrapf(err, "Hostname validation for %s", name)
 	}
 	if !matched {
-		return errors.Errorf("Hostname format mismatch: %s name is not valid", name)
+		return errors.Errorf(`Hostname format mismatch: %s name is not valid.
+			Hostname must have a maximum length of 64 characters,
```

Comment:
Is this function supposed to validate this?

Reply:
Yes it checks against the regex pattern and if the hostname doesn't fit the pattern, then it'll show it as an invalid hostname.

Reply:
Oops sorry didn't push up the new pattern! It should now

Reply:
Alright, can you add a test for this also?

Reply:
Added a test!